backends/concrete-cpu/implementation/Cargo.toml

```diff
@@ -29,6 +29,17 @@ num-complex = { version = "0.4", default-features = false, features = [
 rayon = { version = "1.6", optional = true }
 once_cell = { version = "1.16", optional = true }
 
+tfhe = { version = "0.4", features = [] }
+
+[target.x86_64-unknown-unix-gnu.dependencies]
+tfhe = { version = "0.4", features = ["x86_64-unix"] }
+
+[target.aarch64-unknown-unix-gnu.dependencies]
+tfhe = { version = "0.4", features = ["aarch64-unix"] }
+
+[target.x86_64-pc-windows-gnu.dependencies]
+tfhe = { version = "0.4", features = ["x86_64"] }
+
```

Comment:
Hello if I may (just got back from vacation), the nightly feature does not enable the nightly-avx512 feature of TFHE-rs so on AVX512 CPUs you would not benefit from the AVX512 instructions, you would probably need to do something like so below:

```toml
nightly = ["pulp/nightly", "concrete-fft/nightly", "tfhe/nightly-avx512"]
```

cc @BourgerieQuentin I guess